data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/node/GetHealth.java

```diff
@@ -65,7 +72,7 @@ public GetHealth(final DataProvider provider) {
   public void handleRequest(final RestApiRequest request) throws JsonProcessingException {
     request.header(Header.CACHE_CONTROL, CACHE_NONE);
     if (!chainDataProvider.isStoreAvailable() || syncProvider.getRejectedExecutionCount() > 0) {
-      request.respondWithCode(SC_SERVICE_UNAVAILABLE);
+      request.respondError(SC_SERVICE_UNAVAILABLE, "Node not initialized or having issues");
```

Comment:
nit: Im not sure we need to change this.... not having the error message is fine I think for service_unavailable... 
I'd probably suggest just putting the original 503 response line back in and not having the error here...

Reply:
Ok, will roll back GetHealth and Readiness.